.github/workflows/reusable-update-config.yaml

```diff
@@ -3,10 +3,12 @@ name: Reusable Update Config
 on:
   workflow_call:
     inputs:
-      paths_map:
+      network:
+        required: true
+        type: string
+      files:
```

Comment:
if multiple files are accepted, document the syntax so usage is clear. Otherwise, change variable name to `file`